web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/FileDavResourceBase.java

```diff
@@ -372,7 +252,6 @@ private FileOutVO getFileFromDavFileName(Long id, String fileName) throws Authen
 		} catch (NumberFormatException e) {
 			fileId = null;
 		}
-		// String extension = filePath.getExtension();
 		if (fileId != null && !isTempFile(filePath.getExtension())) {
 			return WebUtil.getServiceLocator().getFileService().getFile(getAuth(), fileId);
 		} else {
```

Comment:
this code is not active, maybe you can complete it and implement WebDAV, so it natively works with MS Word+Excel (also 2003/2007)